cni/network/invoker_cns.go

```diff
@@ -85,13 +86,13 @@ func (invoker *CNSIPAMInvoker) Add(addConfig IPAMAddConfig) (IPAMAddResult, erro
 	}
 
 	info := IPv4ResultInfo{
-		podIPAddress:       response.PodIpInfo.PodIPConfig.IPAddress,
-		ncSubnetPrefix:     response.PodIpInfo.NetworkContainerPrimaryIPConfig.IPSubnet.PrefixLength,
-		ncPrimaryIP:        response.PodIpInfo.NetworkContainerPrimaryIPConfig.IPSubnet.IPAddress,
-		ncGatewayIPAddress: response.PodIpInfo.NetworkContainerPrimaryIPConfig.GatewayIPAddress,
-		hostSubnet:         response.PodIpInfo.HostPrimaryIPInfo.Subnet,
-		hostPrimaryIP:      response.PodIpInfo.HostPrimaryIPInfo.PrimaryIP,
-		hostGateway:        response.PodIpInfo.HostPrimaryIPInfo.Gateway,
+		podIPAddress:       response.PodIpInfo[0].PodIPConfig.IPAddress,
```

Comment:
it's wrong to assume that 0th index will always hold ipv4. You need to check if the IPAddress is v4 family or not

Reply:
This may be able to be changed back to normal or will have to be something different if we end up using the new API. I originally was in the mindset that the first ip would be ipv4 since that is how I originally had it set up as and if there was only one IP it owuld default to the first index anyways. Will still need to be changed though.